New monitoring page (#573)
akeneo/pim-helpcenter: content/md/connectivity/manage-your-connections.md

To be able to view the catalog, you will need to give permissions to your connection user group. [Here is the documentation to help you in this task.](access-rights-on-products.html)
:::

# Monitor a connection

> Once your connection is installed, you'll need to monitor its activity. For this, you can click on the monitoring page directly in the connection navigation menu. 

Reviewer: ```suggestion
Once your connection is installed, you'll need to monitor its activity. To do so, you can click on the monitoring page directly in the connection navigation menu. 
```

To be able to view the catalog, you will need to give permissions to your connection user group. [Here is the documentation to help you in this task.](access-rights-on-products.html)
:::

# Monitor a connection

Once your connection is installed, you'll need to monitor its activity. 

> In other words, you'll probably want to know how many products have been created or updated thanks to your ERP connection, or know how many products have been synchronised via your e-commerce connection **and why a product hasn't been created or updated by a source connection**.

Reviewer 2: ```suggestion
In other words, you'll probably want to know how many products have been created or updated thanks to your ERP connection, or know how many products have been synchronized via your eCommerce connection **and why a product hasn't been created or updated by a source connection**.
```
